src/main/java/org/mickss/atipera/GitHubController.java

```diff
@@ -28,20 +29,20 @@ public GitHubController(RestTemplate restTemplate, ObjectMapper objectMapper) {
     }
 
     @GetMapping("/repos")
-    public ResponseEntity<List<JsonNode>> getRepositories(@RequestParam String username) throws JsonProcessingException {
+    public ResponseEntity<List<RepositoryDTO>> getRepositories(@RequestParam String username) throws JsonProcessingException {
```

Comment:
I think ResponseEntity can be removed at all and we can just return List<RepositoryDTO> ?